### DISCUSSION TITLE
[UserStoriesToRefine] #9 As <user> I want a website that will support fundraising efforts (Patreon, selling logo merchandise, donations, event ticket sales, etc.)

### DISCUSSION THREAD

**Discussion by emecas** [emecas/civic-tech-website-consultancy-pilot-translation#34] [General]
### As <user> I want a website that will support fundraising efforts (Patreon, selling logo merchandise, donations, event ticket sales, etc.)

**Comment by UnitedByChocolate** [comment]
Level 2 - As a cacao producer I can donate a lower dollar amount to support the growth and expansion of United by Chocolate
## Level 3: Acceptance Criteria: 
1. Easy low-dollar value donation button ($2) that accepts a "cash" method (perhaps Venmo)
2. Be able to see the impact of the donation on the total funds raised

Level 2 - As a cacao producer I can see the donation activity so that I feel I can trust how donations are being collected.
## Level 3: Acceptance Criteria: 
1. Be able to see the impact of the donation on the total funds raised
2. Be able to see the designation of the donations


Level 2 - As a supporter I want a way for those that want to support our producer community's education and network expansion to have an easy way to monetarily support.


## Level 3: Acceptance Criteria: 
1. Easy low-dollar value donation button ($2) that accepts a "cash" method (perhaps Venmo)
2. Higher value donation via Patreon
3. "Decide on your own" donation option
